src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -147,6 +147,19 @@ public function register_routes() {
 				'schema'      => array( $this, 'get_public_item_schema' ),
 			)
 		);
+
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/count',
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_count' ),
+					'permission_callback' => array( $this, 'get_count_permissions_check' ),
+				),
+				'schema' => array( $this, 'get_count_schema' ),
```

Comment:
This does absolutely nothing in GET responses. It's only available in the `'help'` context, which is used for OPTIONS requests.

@TimothyBJacobs do you know if it's advisable, or even possible, to have a custom schema for a route here?